packages/scripts/config/webpack.config.js

```diff
@@ -65,16 +81,50 @@ const config = {
 			},
 			{
 				test: /\.svg$/,
+				exclude: /node_modules/,
 				use: [ '@svgr/webpack', 'url-loader' ],
 			},
+			{
+				test: /\.(sc|sa|c)ss$/,
+				exclude: /node_modules/,
+				use: [
+					{
+						loader: MiniCssExtractPlugin.loader,
+					},
+					{
+						loader: require.resolve( 'css-loader' ),
+						options: {
+							sourceMap: ! isProduction,
+						},
+					},
+					{
+						loader: require.resolve( 'postcss-loader' ),
+						options: {
+							ident: 'postcss',
+							plugins: postcssConfig.plugins,
+						},
+					},
+					{
+						loader: require.resolve( 'sass-loader' ),
+						options: {
+							sourceMap: ! isProduction,
+						},
+					},
+				],
+			},
 		],
 	},
 	plugins: [
-		// WP_BUNDLE_ANALYZER global variable enables utility that represents bundle content
-		// as convenient interactive zoomable treemap.
+		// WP_BUNDLE_ANALYZER global variable enables utility that represents
+		// bundle content as convenient interactive zoomable treemap.
 		process.env.WP_BUNDLE_ANALYZER && new BundleAnalyzerPlugin(),
-		// WP_LIVE_RELOAD_PORT global variable changes port on which live reload works
-		// when running watch mode.
+		// MiniCssExtractPlugin to extract the CSS thats gets imported into JavaScript.
+		new MiniCssExtractPlugin( { esModule: false, filename: '[name].css' } ),
+		// MiniCssExtractPlugin creates JavaScript assets for CSS that are
+		// obsolete and should be removed.
```

Comment:
Is there a relevant upstream issue for this? It'd be helpful for future maintenance to know when this workaround can be removed.

And if there's not, should we create one?

Reply:
There is an issue in their repo. I found it but I couldn’t find it again to link, I will try again 😅

Reply:
I think this is the original issue webpack-contrib/mini-css-extract-plugin#151.

Reply:
@Mamaduka, thanks for finding the issue. I included this link in code comment with 
46c2f17